src/controller/java/src/chip/devicecontroller/ChipDeviceController.java

```diff
@@ -626,6 +639,14 @@ public void readPath(
    */
   public native byte[] convertX509CertToMatterCert(byte[] x509Cert);
 
+  /**
+   * Extract skid from paa cert.
+   *
```

Comment:
I don't see this API is called by any Java code, what is this used for?

Reply:
Used to filter paa cert from the paa list according to skid, like this:

```
        for (paa in paaList){
            val skidFromPaaCert = deviceController.extractSkidFromPaaCert(paa)
            if (Arrays.equals(skid, skidFromPaaCert)) {
                return paa
            }
        }
```

Reply:
I can provide an implementation example of AttestationTrustStoreDelegate in java.
But what paa certs can I put?

Reply:
Just use the well-known list certs used in default Trust Store